### DISCUSSION TITLE
render() return TensorXf with values out ouf range [0.0,1.0]

### DISCUSSION THREAD

**Discussion by nebnes** [mitsuba-renderer/mitsuba3#1121] [Q&A]
After rendering, I get a tensorXF with values out of range [0.0,1.0]. 
I get several values above 1.0 around 94.00

It seems out of range values match overly bright pixels. Is it correct to clip the tensor 's high values back to 1.0 ? :
`np.clip(img,0.,1.)`

Is it correct ?

**Comment by merlinND** [comment]
Hello @nebnes,

When using the `hdrfilm`, Mitsuba will produce  _physically based_ renderings with _high dynamic range_. `mi.render` does not apply a tonemapping operation, so there is no particular limit on the value of pixels rendered.

**Reply by merlinND** [reply]
See also @bathal1's answer: https://github.com/mitsuba-renderer/mitsuba-blender/discussions/97#discussioncomment-8953224

@nebnes, please refrain from posting the same question on multiple repos as it creates duplicate work for maintainers. (Or at least, clearly refer to all cross-posts if you do).

**Reply by nebnes** [reply]
Thanks @merlinND for your clear answer. 
And sorry for the duplicate I got lost between the two repo (I thought github was buggy).
